sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceClient.java

```diff
@@ -59,10 +60,12 @@ public WebPubSubClientAccessToken getClientAccessToken(GetClientAccessTokenOptio
             if (options.getExpiresAfter() != null) {
                 requestOptions.addQueryParam("minutesToExpire", String.valueOf(options.getExpiresAfter().toMinutes()));
             }
-            if (CoreUtils.isNullOrEmpty(options.getRoles())) {
-                requestOptions.addQueryParam("role", options.getRoles().stream().collect(Collectors.joining(",")));
+            if (!CoreUtils.isNullOrEmpty(options.getRoles())) {
+                options.getRoles().stream().forEach(roleName -> requestOptions.addQueryParam("role", roleName));
+            }
+            if (!CoreUtils.isNullOrEmpty(options.getGroups())) {
+                options.getGroups().stream().forEach(groupName -> requestOptions.addQueryParam("group", groupName));
             }
-            requestOptions.addQueryParam("api-version", version.getVersion());
```

Comment:
remove adding `api-version` query param because it will be added in`WebPubSubsImpl`, otherwise, `api-version` will appear twice and the call will be failure.
```
https://webpubsubhaoling.webpubsub.azure.com/api/hubs/test/:generateToken?api-version=2022-11-01&api-version=2022-11-01
```